README.md

```diff
@@ -263,55 +264,11 @@ Any configuration options that can be set via the command line can also be speci
     ],
     "cache": true,
     "all": true,
-    "check-coverage": true,
     "report-dir": "./alternative"
   }
 }
 ```
 
-**.nycrc:**
```

Comment:
Removed this section on the `.nycrc` example because it might cause confusion with the `package.json` example right above it. I'm hoping that the explanation above that the configuration can be set in either the `package.json` or `.nycrc` file is enough explanation.